src/models/application-membership.ts

```diff
@@ -225,7 +225,7 @@ const getApplicationMembershipModel = function (
 		 * @function
 		 * @memberof balena.models.application.membership
 		 *
-		 * @description This method adds a user to an application by their username.
+		 * @description This method adds a user to an application by their username if they are a member of the organization.
```

Comment:
How about also adding a check in the function's implementation so that we enforce it right away?
You can also leave a `TODO` to remove that check once we add that check in the API.

This though would block you bumping the SDK in the UI until after the deadline date that we will announce... Was this the reason that you didn't add a runtime check?